rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -180,29 +179,34 @@ class DIETClassifier(EntityExtractor):
         # if true random tokens of the input message will be masked and the model
         # should predict those tokens
         MASKED_LM: False,
-        # if true apply dropout to sparse tensors
-        SPARSE_INPUT_DROPOUT: True,
-        # if true BILOU schema is used for entities
+        # BILOU_flag determines whether to use BILOU tagging or not.
+        # More rigorous however requires more examples per entity
+        # rule of thumb: use only if more than 100 egs. per entity
         BILOU_FLAG: False,
```

Comment:
```suggestion
        BILOU_FLAG: True,
```
I think, True can be default, I don't think it makes thing worse